frontend/app/views/fragments/pricing/priceInfoEvent.scala.html

```diff
@@ -10,20 +10,17 @@
                 <div class="price-info-inline__value qa-event-detail-price">
                     @ticketing.generalReleaseTicketOpt.map(_.priceText)
                 </div>
-	    } { discountTicketing =>
```

Comment:
This change is the same as in master (compare raw view with https://raw.githubusercontent.com/guardian/membership-frontend/master/frontend/app/views/fragments/pricing/priceInfoEvent.scala.html)